BGAnimations/ScreenSelectMusic overlay/PerPlayer/DensityGraph.lua

```diff
@@ -0,0 +1,137 @@
+local player = ...
+local pn = ToEnumShortString(player)
+local p = PlayerNumber:Reverse()[player]
+
+local height = 64
+local width = IsUsingWideScreen() and 286 or 276
+
+-- Max height at 280 BPM 16ths. Anything faster will just scale the graph. 
+local maxNps = 280/60 * 4
```

Comment:
Do you intend to eventually scale all the graphs to 280bpm stream height? I think that might be very discouraging for "casual" players. How about scaling the graphs to the max NPS across all charts in the song? That way the graphs would still be comparable across difficulties, but we would avoid situations where all of the stuff I play only fills the graph height about half way.

Reply:
Yeah the original implementation scaled it to 280 bpm stream height, but I realized the experience wasn't great for the same reason you described. Often times the graph was just completely covered by the breakdown text quad so I got rid of it  (so this is actually dead code at the moment that I forgot to remove). I do like the idea of comparing graphs across all difficulties of a song though.  I think I will implement that and the planned refactor will help with that approach.